[2023-01-26 15:59:03,051] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Leer_Datos_Respuesta_Nasa scheduled__2023-01-01T00:00:00+00:00 [queued]>
[2023-01-26 15:59:03,074] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Leer_Datos_Respuesta_Nasa scheduled__2023-01-01T00:00:00+00:00 [queued]>
[2023-01-26 15:59:03,074] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 15:59:03,075] {taskinstance.py:1377} INFO - Starting attempt 5 of 7
[2023-01-26 15:59:03,075] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 15:59:03,100] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): Leer_Datos_Respuesta_Nasa> on 2023-01-01 00:00:00+00:00
[2023-01-26 15:59:03,108] {standard_task_runner.py:52} INFO - Started process 1156 to run task
[2023-01-26 15:59:03,113] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Respuesta_Nasa', 'Leer_Datos_Respuesta_Nasa', 'scheduled__2023-01-01T00:00:00+00:00', '--job-id', '511', '--raw', '--subdir', 'DAGS_FOLDER/00-ETL_spaceX.py', '--cfg-path', '/tmp/tmpyarlzo6l', '--error-file', '/tmp/tmpslpjkyh9']
[2023-01-26 15:59:03,116] {standard_task_runner.py:80} INFO - Job 511: Subtask Leer_Datos_Respuesta_Nasa
[2023-01-26 15:59:03,203] {task_command.py:371} INFO - Running <TaskInstance: Respuesta_Nasa.Leer_Datos_Respuesta_Nasa scheduled__2023-01-01T00:00:00+00:00 [running]> on host fd9891b969b0
[2023-01-26 15:59:03,305] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=Respuesta_Nasa
AIRFLOW_CTX_TASK_ID=Leer_Datos_Respuesta_Nasa
AIRFLOW_CTX_EXECUTION_DATE=2023-01-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=5
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-01T00:00:00+00:00
[2023-01-26 15:59:03,307] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-01-26 15:59:03,307] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /tmp && head /tmp/response_20230101.txt']
[2023-01-26 15:59:03,322] {subprocess.py:85} INFO - Output:
[2023-01-26 15:59:03,330] {subprocess.py:92} INFO - ***tmplz0uzku1
[2023-01-26 15:59:03,330] {subprocess.py:92} INFO - ***tmpukzfi137
[2023-01-26 15:59:03,330] {subprocess.py:92} INFO - platzi_data_20210101.csv
[2023-01-26 15:59:03,330] {subprocess.py:92} INFO - platzi_data_20230101.csv
[2023-01-26 15:59:03,330] {subprocess.py:92} INFO - platzi_data_20230102.csv
[2023-01-26 15:59:03,331] {subprocess.py:92} INFO - platzi_data_20230103.csv
[2023-01-26 15:59:03,331] {subprocess.py:92} INFO - platzi_data_{{ds_nodash}}.csv
[2023-01-26 15:59:03,331] {subprocess.py:92} INFO - pymp-183zsj8a
[2023-01-26 15:59:03,331] {subprocess.py:92} INFO - pymp-_z20wx4h
[2023-01-26 15:59:03,331] {subprocess.py:92} INFO - pymp-igsi0xnb
[2023-01-26 15:59:03,331] {subprocess.py:92} INFO - pymp-w4p836yo
[2023-01-26 15:59:03,331] {subprocess.py:92} INFO - response_20230101.txt
[2023-01-26 15:59:03,331] {subprocess.py:92} INFO - response_20230102.txt
[2023-01-26 15:59:03,332] {subprocess.py:92} INFO - response_20230103.txt
[2023-01-26 15:59:03,332] {subprocess.py:92} INFO - spacex_20230101.json
[2023-01-26 15:59:03,332] {subprocess.py:92} INFO - spacex_20230102.json
[2023-01-26 15:59:03,332] {subprocess.py:92} INFO - spacex_20230103.json
[2023-01-26 15:59:03,332] {subprocess.py:92} INFO - tmpf08jxwfi
[2023-01-26 15:59:03,332] {subprocess.py:92} INFO - tmpnvc2r2ky
[2023-01-26 15:59:03,332] {subprocess.py:92} INFO - tmpqhhk7hje
[2023-01-26 15:59:03,332] {subprocess.py:92} INFO - tmpslpjkyh9
[2023-01-26 15:59:03,333] {subprocess.py:92} INFO - Confirmaci√≥n de la NASA, pueden proceder
[2023-01-26 15:59:03,333] {subprocess.py:96} INFO - Command exited with return code 0
[2023-01-26 15:59:03,379] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Respuesta_Nasa, task_id=Leer_Datos_Respuesta_Nasa, execution_date=20230101T000000, start_date=20230126T155903, end_date=20230126T155903
[2023-01-26 15:59:03,444] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-26 15:59:03,598] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
